content/en/tracing/services/service_page.md

```diff
@@ -42,6 +43,32 @@ Consult on this page:
+ 
+## Service health
+
+{{< callout header="Opt in to the private beta!" url="https://docs.google.com/forms/d/1KsC7DPKBJ3K0wP67fK50JhRRX-CseP4FUrFrOvCYbQM/viewform?edit_requested=true" >}}
+  Service health is in private beta. To request access, complete the form.
+{{< /callout >}} 
+
+The **Service Health** panel automatically provides a real-time summary of service signals to help you understand if a service needs your attention.
+
+Service health factors in many types of signals, including monitors, incidents, watchdog insights, and error tracking issues, and surfaces the most critical alerts. Additionally, the panel provides links to associated incidents so you can take action, as necessary.
+
+{{< img src="/tracing/services/service_page/service-health.png" alt="Service Health panel on service page showing an active incident." style="width:100%;" >}}
+
+To access service health:
+
+1. Go to [APM > Service Catalog][23].
+2. Hover over a service and click **Full Page**.
+3. Select **Service Health**.
+
+The health of your service can have a *Ok*, *Warning*, or *Alert* status. Statuses are determined based on the following conditions and signal types:
+
+|   Status    |                         Condition                          |                                                            Monitors                                                            |            Incidents            |                        Watchdog Insights                        |          Error Tracking Issues          |
+|:-----------:|:----------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------:|:---------------------------------------------------------------:|:---------------------------------------:|
+|  **Alert**  |               At least one signal is active.               |               Non-muted alerting P1 monitor or monitor with a paging integration setup (PagerDuty or Opsgenie).                | Active incident of any severity | Active faulty deployment, ongoing APM latency/error rate alert. |                   --                    |
+| **Warning** | At least one signal is active and not in a critical state. | Non-muted alerting P2 monitor, warning P1 monitor, or warning monitor with a paging integration setup (PagerDuty or Opsgenie). | Stable incident of any severity |                   Ongoing log anomaly alert.                    | New (within 48 hours) issues for review |
+| **Ok** |    No signal from critical or warning state is active.     |                                                               --                                                               |               --                |                               --                                |                   --                    |
```

Comment:
I'm having a lot of trouble understanding what's happening in this table. The headers don't seem lined up in a way that makes sense. If you get a monitors alert, it means "Non-muted alerting P1 monitor or monitor with a paging integration setup (PagerDuty or Opsgenie)."? What's the verb there? I also can't tell how the "condition" column is meant to be interpreted. 

Basically I think this table needs to be reformatted, and (I think) each statement within needs to have a clear subject and verb (for example, saying "An incident of any severity is active" vs. "Active incident of any severity")—but I also don't have enough context (or understanding of the table) to be sure about this